index.html

```diff
@@ -94,14 +96,39 @@ <h3>Portfolio Item 6</h3>
 		</section>
 	</section>
 
-
 	<!-- \/ for future use by contact \/ -->
 
 	<section class="contact">
 		<section class="container">
-			<div id="contact-buttons-box">
-				
-			</div>
+
+			<h2>Contact Me</h2>
+				<article id="contact-mail">
+					<a href="mailto:jdselby24@mgail.com">
+						<i class="fas fa-envelope"></i>
+						<p>Email</p>
+					</a>
+				</article>
+
+				<article id="contact-linked">
+					<a href="https://www.linkedin.com/in/josh-selby-607182185/">
```

Comment:
Linked in one directs you to a sign-in page if you don't have an account. Not all employers will use this so perhaps not as useful?